### DISCUSSION TITLE
'closeOnOutsideClick' not working??

### DISCUSSION THREAD

**Discussion by JBRDiver** [biati-digital/glightbox#375] [Q&A]
I LOVE gLightbox but am totally stumped - please help!

I'd like the visitor to be able to close the current slide and return to the gallery of images by simply clicking outside of the current image. I've used the following code but the 'click outside' parameter seems to be having no effect????

var lightbox = GLightbox({
      selector: 'gallery-lightbox',
      closeOnOutsideClick: 'true'
    });

Am I doing something obviously wrong here?

Here's the page in question...

https://www.vobster.com/gallery-viewer.php?gallery=20221007-glowswim2022&base=gallery-events&return=gallery-events.php

Thank you in advance! :-)

**Comment by gingerchew** [comment]
Hmmm that is odd, maybe this issue can help? https://github.com/biati-digital/glightbox/issues/188
